### ISSUE TITLE
Copy and pasted text gets wrapped in span with font-size style

### ISSUE THREAD

**Issue by fuzl-llc** [OrchardCMS/OrchardCore#9655]
### Describe the bug

If you type a sentence in an HtmlBodyPart with the WYSIWYG editor or a Flow page's "Paragraph" widget and then copy and paste it, the pasted text gets wrapped in a span tag with the font-size style set to 1rem.  This results in the original sentence font being larger than the rest.  It is also not good markup to have inline styles on things, especially since you might not even notice (as I didn't for a while).

### To Reproduce
1. Create a new site with the blog theme
2. Create a Page content item that includes the flow part
3. Add a paragraph widget
4. Type "This is a test."
5. Copy and paste that whole text so you have "This is a test.This is a test."
6. Click the Preview button
6. Notice the second "This is a test." has a smaller font and the markup for it has an unexpected span tag:

`<p>This is a test.<span style="font-size: 1rem">This is a test.</span><br></p>`

You can also repro using a new "Html Page" content type that has just autoroute, title, and html part (with the WYSIWYG editor selected for html part).

### Expected behavior

We get just simple HTML, e.g.:

`<p>This is a test.This is a test.</p>`

**Comment by Skrypt** [comment]
If you want to copy only the text you need to use CTRL+SHIFT+V. This will paste the selected text as plain text.
Also, you can use the Trumbowyg editor which has an option for this as stated in their documentation : 
https://alex-d.github.io/Trumbowyg/documentation/#remove-format-pasted

**Comment by fuzl-llc** [comment]
Thanks for the info. I think this might be a slightly different issue though. I can understand if I copy formatted text from Word or elsewhere and those formats come along by default when I paste with standard Ctrl + V.  In that case Ctrl + Shift + V seems to be the universally used solution to get a "plain text" paste. However, in this case I'm _copying_ plain text that I just typed and when I paste it has formatting that seems to have come from nowhere. In the case above, I type "This is a test." which results in this HTML:

`<p>This is a test.</p>`

I then copy that text, which has no formatting related to font size, and paste it, resulting in this HTML:

`<p>This is a test.<span style="font-size: 1rem">This is a test.</span><br></p>`

So, the span with font-size formatting didn't come from the text I copied as in the Word example where I actually copied formatted text. Where does the formatting style tag come from and wouldn't that be a bug? I guess maybe it isn't Orchard's bug but maybe there is a fix that can be put in place.
